tests/system/marathon_common_tests.py

```diff
@@ -1007,6 +1007,31 @@ def test_metric_endpoint(marathon_service_name):
         "service.mesosphere.marathon.app.count is absent"
 
 
+def test_healtchcheck_and_volume():
```

Comment:
Oh, and could you please put a comment here that's this is a regression test linking the original JIRA issue? Thx!

Reply:
I tried to find the original JIRA, but I could not find it :( Karsten raised the same question here https://phabricator.mesosphere.com/D612 but didn`t got an answer :(

The commit fixed the behavior was reviewed here https://phabricator.mesosphere.com/D606

Reply:
a better name then? smth. that describes what is being tested e.g. `test_healthcheck_with_user_network_and_volumes`?